examples/plot_firing_pattern.py

```diff
@@ -30,26 +29,33 @@
 import matplotlib.pyplot as plt
 
 net = Network(params)
-with NetworkBuilder(net) as network_builder:
-    network_builder.cells[0].plot_voltage()
-
-    # The cells are stored in the network object as a list
-    cells = network_builder.cells
-    print(cells[:5])
-
-    # We have different kinds of cells with different cell IDs (gids)
-    gids = [0, 35, 135, 170]
-    for gid in gids:
-        print(cells[gid].name)
-
-    # We can plot the firing pattern of individual cells
-    network_builder.cells[0].plot_voltage()
-    plt.title('%s (gid=%d)' % (cells[0].name, gid))
+dpls = simulate_dipole(net, n_trials=1)
+
+# The cell IDs (gids) are stored in the network object as a dictionary
+gid_dict = net.gid_dict
+print(net.gid_dict)
+
+# Simulated cell responses are stored in the Spikes object as a dictionary.
+trial_idx = 0
+vsoma = net.spikes.vsoma[trial_idx]
+print(vsoma.keys())
+
+# We can plot the firing pattern of individual cells by indexing with the gid
+gid = 170
+t_vec = net.spikes.t_vec[trial_idx]
+plt.figure(figsize=(4, 4))
+plt.plot(t_vec, vsoma[gid])
+plt.title('%s (gid=%d)' % (net.gid_to_type(gid), gid))
+plt.xlabel('Time (ms)')
+plt.ylabel('Voltage (mV)')
+plt.show()
```

Comment:
do you also want to plot the spikes in another subplot for one cell? We already have the function. Got the idea from your comment in the test :)

Reply:
May as well do the raster plot since it is the "firing pattern" example.